daemon/daemon.go

```diff
@@ -760,9 +760,23 @@ func (daemon *Daemon) Shutdown() error {
 
 				go func() {
 					defer group.Done()
-					// If container failed to exit in 10 seconds of SIGTERM, then using the force
-					if err := c.Stop(10); err != nil {
-						logrus.Errorf("Stop container %s with error: %v", c.ID, err)
+					if c.IsPaused() {
```

Comment:
@coolljt0725 sorry for late response, I meant that for now, we should probably handle this whole PR (every addition from `if c.IsPaused() {`) in a function so that the windows version keeps the old behavior for now (with maybe a `TODO(windows): Handle docker restart with paused containers` added)

Doest that make more sense?